tests/oval_details/test_oval_details.sh

```diff
@@ -10,13 +10,37 @@ output_dir=`mktemp -d -t oval_details_XXXXXX`
 
 # Test cases.
 
-function test_oval_details {
+function test_oval_details_implicit {
+  # Tests if OVAL Details are present in HTML report by default
+  # without specifing --oval-results --results (new behavior)
   xccdffile=$srcdir/$1.xccdf.xml
   resultfile=$output_dir/$1.result.xml
   reportfile=$output_dir/$1.report.html
+  oval_resultfile="$1.oval.xml.result.xml"
+  $OSCAP xccdf eval --report $reportfile $xccdffile
+  [ -f $reportfile ]
+  [ ! -f $resultfile ]
```

Comment:
I would add one more test here to also check if the results file is not generated in the CWD, just to be sure: `[ ! -f $(basename $resultfile) ]`

Reply:
OK, I'll do that.